kas/kas-poky-meson.yml

```diff
@@ -0,0 +1,54 @@
+header:
+  version: 8
+
+machine: amediatech-x96-max
+distro: poky
+target:
+  - amlogic-image-headless-sd
+
+repos:
+  meta-raspberry:
```

Comment:
Why is there reference to meta-raspberry ? (Perhaps it’s needed, no worry then)

Reply:
Oh, that was a copy paste. I'll fix it.